data/pages-content/terms.md

```diff
@@ -5,113 +5,105 @@ title: Terms and Conditions
 
 For debit or credit card payments, the cards will be charged immediately
 following the application. Debit and credit card payments are handled by the
-service provider [Stripe](https://stripe.com). The Organizers do not store any
+service provider [Stripe](https://stripe.com). The Organisers do not store any
 card information.
 
 ## 3. Performance and Cancellation
 
 Admission to the Event is granted based on tickets (“Event Tickets” or
 “Conference Passes”). The ticket fee listed on the Event Website covers the
-admission of the Event (or part thereof) to the extent defined on the Event
+admission to the Event (or part thereof) to the extent defined on the Event
 Website for each ticket category. The same applies to any additional items which
-may be sold via the Event Website, such as e.g. SIM cards, tickets for museums,
+may be sold as part of the Event, such as e.g. SIM cards, tickets for museums,
 local transport, etc.
 
-Changes to the program, the event content, schedule and the speakers may occur
-due to organizational reasons and are not considered as a breach of the purchase
+Changes to the programme, the event content, schedule and the speakers may occur
+due to organisational reasons and are not considered as a breach of the purchase
 contract. There is no right to refunds for any unused services which were
 properly offered.
 
-The Organizers are entitled to withdraw from the contract if carrying out the
+The Organisers are entitled to withdraw from the contract if carrying out the
 Event would be uneconomic (e.g. due to insufficient number of exhibitors,
-sponsors or participants). The same applies if the Event is canceled, in whole
+sponsors or participants). The same applies if the Event is cancelled, in whole
 or in part, for reasons beyond their reasonable control, such as act of God,
-labor disputes, civil commotion, pandemics, epidemics, terrorism, vandalism,
+labour disputes, civil commotion, pandemics, epidemics, terrorism, vandalism,
 war, lack of power or utilities, floods, fire, acts of Government or regulatory
 authorities or other events of such effect that carrying out or the continuation
-of the event would be unreasonable in the circumstances. The Organizers are
-under no obligation to reorganize the event.
+of the event would be unreasonable in the circumstances. The Organisers are
+under no obligation to reorganise the event.
 
 In case of a cancellation of the Event, all participants will be notified by the
-Organizers without delay and any payments made will be refunded. The Organizers’
-liability is limited to refund of the participation fee. All other liability for
+Organisers without delay and any payments made will be refunded. The Organisers’
+liability is limited to refunding the participation fee. All other liability for
 participant claims is excluded except in cases of intent or gross negligence on
```

Comment:
I think "willful" is the word, rather than "intent", which doesn't make any sense in my ears.